### PR TITLE
WIP Feature/client app refactoring

### PR DESCRIPTION
**Pull request by mckrava**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```
Comment (jak-pan) [inline LICENSE:1]: Re-add this back

**Review by jak-pan — COMMENTED** [review]
Remove Vue assets from public folder. If we want to keep all of the icons regenerate them using https://github.com/galacticcouncil/HydraDX-assets

File: src/assets/styles/scss/abstracts/_breakpoints.scss [hunk 2]
```diff
@@ -0,0 +1,123 @@
+// Breakpoint viewport sizes and media queries.
```
Comment (jak-pan) [inline src/assets/styles/scss/abstracts/_breakpoints.scss:1]: We are probably not using this, are we?

File: src/assets/styles/scss/abstracts/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
+$hdx-bgr-gray: #f5f6fa;
```
Comment (jak-pan) [inline src/assets/styles/scss/abstracts/_variables.scss:1]: We are probably not using this are we?

File: src/assets/styles/scss/base/_typography.scss [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  color: #aaa;
+  border-color: #aaa;
+}
+
+.disabled {
+  color: #aaa;
+  border-color: #aaa;
+}
+
+.noPools,
```
Comment (jak-pan) [inline src/assets/styles/scss/base/_typography.scss:77]: This probably doesn't belong here?

File: src/components/PoolList.vue [hunk 5]
```diff
@@ -3,12 +3,18 @@
 </template>
 
 <script lang="ts">
-import Vue from "vue";
-import { mapGetters } from "vuex";
+// import Vue from "../vue-typed/vue-typed";
```
Comment (jak-pan) [inline src/components/PoolList.vue:6]: Do we want to keep these?

File: src/layouts/AppLayout.vue [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+import { useStore } from '@/store';
+
+export default defineComponent({
+  name: 'AppLayout',
+  setup() {
+    const { dispatch } = useStore();
+
+    dispatch('initializeApiSMGeneral');
+  },
+  // setup() {
```
Comment (jak-pan) [inline src/layouts/AppLayout.vue:24]: What about this?

File: src/store/general/index.ts [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import { Module } from 'vuex';
+
+import { state } from '@/store/general/state.ts';
+import { mutations } from '@/store/general/mutations.ts';
+import { actions } from '@/store/general/actions.ts';
+import { getters } from '@/store/general/getters.ts';
+
+export const general: Module<GeneralState, MergedState> = {
+  // namespaced: true,
```
Comment (jak-pan) [inline src/store/general/index.ts:9]: remove?

File: src/store/trade/mutations.ts [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+  },
+  UPDATE_TRANSACTIONS__TRADE(state, transaction) {
+    if (transaction.id != null) {
+      let transactionData;
+      if (transaction.index != null) {
+        transactionData = { ...state.unpairedTransactions[transaction.index] };
+
+        //TODO Should be enough with new reactivity logic in Vue 3
+
+        //Vue.delete(state.unpairedTransactions, transaction.index);
```
Comment (jak-pan) [inline src/store/trade/mutations.ts:32]: This might be the reason of duplication?

Comment (jak-pan) [reply src/store/trade/mutations.ts:32]: Or it might have been commented out before?

**Comment by jak-pan** [comment]
pendingAction doesn't work. It looks to me like it should be global thing but still it should work at least in the pool. Right now it doesn't get applied.

File: LICENSE (hunk 1 above)
Comment (mckrava) [reply LICENSE:1]: fixed

File: src/assets/styles/scss/abstracts/_breakpoints.scss (hunk 2 above)
Comment (mckrava) [reply src/assets/styles/scss/abstracts/_breakpoints.scss:1]: fixed

File: src/assets/styles/scss/abstracts/_variables.scss (hunk 3 above)
Comment (mckrava) [reply src/assets/styles/scss/abstracts/_variables.scss:1]: fixed

File: src/assets/styles/scss/base/_typography.scss (hunk 4 above)
Comment (mckrava) [reply src/assets/styles/scss/base/_typography.scss:77]: fixed

File: src/components/PoolList.vue (hunk 5 above)
Comment (mckrava) [reply src/components/PoolList.vue:6]: fixed

File: src/layouts/AppLayout.vue (hunk 6 above)
Comment (mckrava) [reply src/layouts/AppLayout.vue:24]: fixed

File: src/store/general/index.ts (hunk 7 above)
Comment (mckrava) [reply src/store/general/index.ts:9]: fixed

File: src/store/trade/mutations.ts (hunk 8 above)
Comment (mckrava) [reply src/store/trade/mutations.ts:32]: Will work with this. Need more debug.

**Comment by mckrava** [comment]
> pendingAction doesn't work. It looks to me like it should be global thing but still it should work at least in the pool. Right now it doesn't get applied to the UI

fixed

**Review by jak-pan — APPROVED** [review]

File: src/store/trade/mutations.ts (hunk 8 above)
Comment (jak-pan) [reply src/store/trade/mutations.ts:32]: Create new issue pls

**Pull request merged**